client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.tsx

```diff
@@ -58,10 +60,20 @@ const PatternListRenderer = ( {
 	shownPatterns,
 	selectedPattern,
 	activeClassName,
+	emptyPatternText,
 	onSelect,
 }: PatternListRendererProps ) => {
 	return (
 		<>
+			{ emptyPatternText && (
```

Comment:
Display the empty pattern for the new layout so that users are able to remove the selected header/footer

![image](https://user-images.githubusercontent.com/13596067/220595403-e3bab970-59db-4bbd-81ff-24523c09316b.png)